cli/pcluster/config/mappings.py

```diff
@@ -128,9 +128,10 @@
             r"([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])"
             r"(\/([0-9]|[1-2][0-9]|3[0-2]))$",
     "efs_fs_id": r"^fs-[0-9a-z]{8}$|^fs-[0-9a-z]{17}|NONE$",
-    "file_path": r"\/?[\w:]+",
+    "file_path": r"^\/?[^\/.\\][^\/\\]*(\/[^\/.\\][^\/]*)*$",
+    "queue_settings": r"^[a-zA-Z][a-zA-Z0-9-]{0,29}(,\s*[a-zA-Z][a-zA-Z0-9-]{0,29})*$",
     "fsx_fs_id": r"^fs-[0-9a-z]{17}|NONE$",
-    "greater_than_25": r"^([0-9]+[0-9]{2}|[3-9][0-9]|2[5-9])$",
+    "greater_than_25": r"^([1-9][0-9]*[0-9]{2}|[3-9][0-9]|2[5-9])$",
```

Comment:
if you fixed the regex let's add a test to cover the wrong case that we were supporting before.

Reply:
From testing a bit further, my change did not fix anything as the number is apparently taken as an int (which erases the leading 0) and then compared using this regex. This means that my change is not useful since leading zeros will not be there. Should I delete it or just let it as is, as it seems more logical when reading?
This regex would still allow 025 as 025 would first be rendered to 25 and then passed to the regex. The regex would thus not match what is expected if we let my change.
Testing this with leading zeros would thus not be useful as they would not appear anyway.

Reply:
As it is a useless change, we let as it was before